easybuild/easyconfigs/f/FLTK/FLTK-1.3.5-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,41 @@
+#
+# author: Dina Mahmoud Ibrahim ( Cairo University )
+#
+easyblock = 'ConfigureMake'
+
+name = 'FLTK'
+version = '1.3.5'
+
+homepage = 'http://www.fltk.org'
+description = """FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,
+ and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL
+ and its built-in GLUT emulation."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
```

Comment:
I suspect you have a good reason, but why not GCCcore?

Reply:
For the simple reason that this is getting into the GUI performance realm. And for that i prefer the upper level compiler when using intel based toolchains.